### PR TITLE
various

### PR DESCRIPTION
**Pull request by MBoretto**
Doc block, various error, fixed thanks to incredible new tools..

### CODE REVIEW

**Comment by noplanman** [comment]
Good to merge, mostly DocBlock changes and some code changes that don't affect any behaviour.

👍

File: src/Commands/AdminCommands/SendtochannelCommand.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ class SendtochannelCommand extends AdminCommand
     /**
      * Conversation Object
      *
-     * @var Longman\TelegramBot\Conversation
+     * @var \Longman\TelegramBot\Conversation
```
Comment (noplanman) [inline src/Commands/AdminCommands/SendtochannelCommand.php]: @MBoretto Did you deliberately leave all full namespaces as opposed to simplified ones?

PhpStorm automatically prefers simplified ones for me.

@akalongman @jacklul 
Asking you guys too, which standard shall we go by?
```php
@var \Longman\TelegramBot\Conversation
```
or
```php
@var Conversation
```
(this obviously implies that we have a `use Longman\TelegramBot\Conversation` at the top.

**Comment by MBoretto** [comment]
I add it because with the \ in the front and the full namespace php storm seems able to recognise the type of the variable

**Comment by noplanman** [comment]
Interesting, it seems to work for me without the whole namespace if there is an appropriate `use` present.
So you would suggest to add the full namespace to **all** docblock variables?

**Pull request merged**